x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/rest/SecurityRestFilterTests.java

```diff
@@ -48,7 +49,9 @@
 
 public class SecurityRestFilterTests extends ESTestCase {
```

Comment:
Should we add a test here to verify we process secondary authentication failures correctly?